flows/interfaces.go

```diff
@@ -163,12 +163,17 @@ type Context interface {
 }
 
 type Event interface {
-	CreatedOn() *time.Time
+	CreatedOn() time.Time
 	SetCreatedOn(time.Time)
 
 	Step() StepUUID
 	SetStep(StepUUID)
 
+	FromCaller() bool
+	SetFromCaller(bool)
```

Comment:
We don't currently know/need that but is that something you think we should log?